sdk/storage/Azure.Storage.Blobs/tests/MockKeyEncryptionKey.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections;
+using System.Security.Cryptography;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core.Cryptography;
+
+namespace Azure.Storage.Blobs.Tests
+{
+    /// <summary>
+    /// Mock for a key encryption key. Not meant for production use.
+    /// </summary>
+    internal class MockKeyEncryptionKey : IKeyEncryptionKey, IKeyEncryptionKeyResolver
```

Comment:
Have we tested this against KeyVault's Track 2 library?  We should really get that checked in somewhere so we're validating the only meaningful customer scenario end to end.

Reply:
Have we tested this against Track 1 for that matter?  Or other languages?  It'd be great to have confidence in the interop story before we ship.

Reply:
We have not tested against keyvault yet as it was still under development when writing this.

We have not run backwards compatible tests yet, though that is a requirement before shipping. Unfortunately, since this touches files outside of the crypto package, we need to get those changes in before main package release, even if we delay the crypto package release to accommodate those tests.